packages/rpc/src/client.ts

```diff
@@ -27,46 +31,86 @@ import type {
 	SetUserVoiceSettingsData,
 	RelationShip,
 } from './typings/structs';
-import { RPCCommands, RPCEvents, type LobbyType, RelationshipType } from './typings/types';
-import { pid as getPid, uuid } from './util';
-
-function subKey(event: string, args?: unknown[]) {
-	return `${event}${JSON.stringify(args)}`;
-}
+import {
+	RPCCommands,
+	RPCEvents,
+	type LobbyType,
+	RelationshipType,
+	RPCErrorCodes,
+	PromptType,
+} from './typings/types.js';
+import { pid as getPid } from './util.js';
 
 type Class<T extends new (...args: any[]) => unknown> = T extends new (...args: any[]) => infer R ? R : never;
 
+const MAX_TIME_RANGE = 2_147_483_647_000;
+
 export interface RPCClientOptions {
-	transport?: keyof typeof transports;
+	/**
+	 * Origin link specified in the [Portal](https://discord.com/developers/applications)
+	 */
 	origin?: string;
+	/**
+	 * rpc transport
+	 */
+	transport: keyof typeof transports;
 }
 
 export interface RPCLoginOptions {
+	/**
+	 * Access token
+	 */
+	accessToken: string;
+	/**
+	 * Client ID
+	 */
 	clientId: string;
+	/**
+	 * Client secret
+	 */
 	clientSecret: string;
-	accessToken: string;
-	rpcToken: string;
-	tokenEndpoint: string;
+	prompt: PromptType;
```

Comment:
Missing tsdoc

Reply:
I am not really good with writing docs, is it possible for other contributor to work on documentation?